tests/readme.md

```diff
@@ -46,12 +61,12 @@ Philosophically, regression tests treat the "past as truth" - the correct output
 
 ## Test data
 
-Data needed for testing should be placed in `tests/data`. This should be limited to small text files e.g. `.csv`, `.json`, `.yml`. Binary data should be stored outside the repository and referenced e.g. using a DOI. Discuss more complex test data requirements for your **pull request** in the motivating **issue**.
+Data needed for testing should be placed in `tests/data`. This should be limited to small files (current limit: 300KB). Larger data files should be downloaded from external sources to `tests/test_data/download`, which will be kept locally but not uploaded. In future, we may have the external sources referenced, e.g. using a DOI. Discuss more complex test data requirements for your **pull request** in the motivating **issue**.
```

Comment:
Do we need to be clear that the download cannot be a manual process i.e. that the test code needs to acquire the data it needs?

Perhaps `In future, we may have the external sources referenced, e.g. using a DOI.` can be removed?